docs/disco.sh

```diff
@@ -0,0 +1,200 @@
+trap 'exit' ERR
+
+export GCLOUD_ZONE="us-east1-c"
```

Comment:
This should be configurable, people do it (https://github.com/hammerlab/stratocumulus/pull/38)